e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    let config;
+
+    before(() => {
+        // Get config
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
+
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
```

Comment:
Please follow the same before/after approach in [your other PR](https://github.com/mattermost/mattermost-webapp/pull/4330/files#diff-e8e5668a9849acec0e1a98afb29a0883R11-R29) and deal with the conflicts later once we merged whichever.